guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -34,8 +34,9 @@ Ensure that you understand the following terms:
 
 To clone {ProjectServer}, ensure that you have the following resources available:
 
-* A minimal install of {EL} 7 to become the target server.
-Do not install {EL} 7 software groups, or third-party applications.
+* A minimal install of {EL} 8 to become the target server.
+Do not install {EL} 8 software groups, or third-party applications.
+
```

Comment:
```suggestion
```

Any reason for that empty line?

Reply:
That shouldn't be there. Must have happened when restoring the deleted doc file. Some things had to be edited again. Mistake fixed.